paasta_tools/paasta_metastatus.py

```diff
@@ -327,7 +333,11 @@ def print_output(argv: Optional[Sequence[str]] = None) -> None:
     args = parse_args(argv)
 
     system_paasta_config = load_system_paasta_config()
-
+    service_config_dict = load_kubernetes_service_config_no_cache(
```

Comment:
this should probably be using `load_kubernetes_service_config`, which will avoid re-parsing YAML if we happen to load the same config again within a short period of time.